### PR TITLE
CapabilityStatements: Add in supporting resources

### PR DESCRIPTION
**Pull request by dt-r**
To provide clarity the support expectations for any particular profile:
* a new section has been added to the CapabilityStatements identifying where a profile has a Must Support reference to other profiles or resources (i.e., Target Profiles).
* the SHOULD resources have been added (i.e. those directly referenced from ServiceRequest & part of the agreed Scaffold)
* the MAY resources have been added (i.e. those that are referenced from a SHOULD profile)

### CODE REVIEW

File: input/pagecontent/CapabilityStatement-au-erequesting-placer-intro.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### Summary of Must Support and References Between Profiles
+The table below provides identifies where a Profile has a Must Support reference to other profiles or resources (i.e., Target Profiles). This list is provided to clarify where additional profiles or resources may need to be supported if a placer supports a particular resource.
```
Comment (dbojicic) [inline input/pagecontent/CapabilityStatement-au-erequesting-placer-intro.md:2]: remove 'provides'

File: input/pagecontent/CapabilityStatement-au-erequesting-server-intro.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+### Summary of Must Support and References Between Profiles
+The table below provides identifies where a Profile has a Must Support reference to other profiles or resources (i.e., Target Profiles). This list is provided to clarify where additional profiles or resources may need to be supported if a server supports a particular resource.
```
Comment (dbojicic) [inline input/pagecontent/CapabilityStatement-au-erequesting-server-intro.md:2]: remove provides

File: input/pagecontent/CapabilityStatement-au-erequesting-server-intro.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+|Observation|[Observation](http://hl7.org/fhir/R4/observation.html)|-|
+|Patient|[AU Core Patient](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html)|-|
+|Practitioner|[AU Core Practitioner](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html)|-|
+|PractitionerRole|[AU Core PractitionerRole](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html)|[AU Core Organization](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-organization.html), [AU Core Practitioner](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html)|
+|Procedure|[AU Core Procedure](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-procedure.html)|[AU Core Condition](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-condition.html), [Observation](http://hl7.org/fhir/R4/observation.html), [AU Core Procedure](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-procedure.html), [DocumentReference](http://hl7.org/fhir/R4/documentreference.html)|
```
Comment (dbojicic) [inline input/pagecontent/CapabilityStatement-au-erequesting-server-intro.md:16]: Missing: AU Core Patient, AU Core Encounter

File: input/pagecontent/CapabilityStatement-au-erequesting-placer-intro.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+|Observation|[Observation](http://hl7.org/fhir/R4/observation.html)|-|
+|Patient|[AU Core Patient](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html)|-|
+|Practitioner|[AU Core Practitioner](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html)|-|
+|PractitionerRole|[AU Core PractitionerRole](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html)|[AU Core Organization](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-organization.html), [AU Core Practitioner](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html)|
+|Procedure|[AU Core Procedure](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-procedure.html)|[AU Core Condition](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-condition.html), [Observation](http://hl7.org/fhir/R4/observation.html), [AU Core Procedure](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-procedure.html), [DocumentReference](http://hl7.org/fhir/R4/documentreference.html)|
```
Comment (dbojicic) [inline input/pagecontent/CapabilityStatement-au-erequesting-placer-intro.md:16]: Missing: AU Core Patient, AU Core Encounter

File: input/resources/au-erequesting-placer.xml [hunk 5]
```diff
@@ -83,5 +83,252 @@
...
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="MAY"/>
+      </extension>
+      <type value="Condition"/>
+      <supportedProfile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-condition">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHALL"/>
+        </extension>
+      </supportedProfile>
+      <documentation value="Placers **MAY** support the Condition resource, the AU Core profile, and the conformance expectations for the PractitionerRole resource."/>
```
Comment (dbojicic) [inline input/resources/au-erequesting-placer.xml:211]: Typo "...and the conformance expectations for the PractitionerRole resource", should say "..and the conformance expectations for the Condition resource."

File: input/resources/au-erequesting-server.xml [hunk 6]
```diff
@@ -105,5 +105,252 @@
...
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="MAY"/>
+      </extension>
+      <type value="Condition"/>
+      <supportedProfile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-condition">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHALL"/>
+        </extension>
+      </supportedProfile>
+      <documentation value="Servers **MAY** support the Condition resource, the AU Core profile, and the conformance expectations for the PractitionerRole resource."/>
```
Comment (dbojicic) [inline input/resources/au-erequesting-server.xml:233]: Typo "...and the conformance expectations for the PractitionerRole resource", should say "..and the conformance expectations for the Condition resource."

File: input/resources/au-erequesting-placer.xml [hunk 7]
```diff
@@ -83,5 +83,252 @@
...
+          <valueCode value="SHOULD"/>
+        </extension>
+      </referencePolicy> -->
+    </resource>
+    <!-- RELATEDPERSON -->
+    <resource>
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="MAY"/>
+      </extension>
+      <type value="Procedure"/>
```
Comment (dbojicic) [inline input/resources/au-erequesting-placer.xml:319]: Should be "RelatedPerson"

File: input/resources/au-erequesting-server.xml [hunk 8]
```diff
@@ -105,5 +105,252 @@
...
+          <valueCode value="SHOULD"/>
+        </extension>
+      </referencePolicy> -->
+    </resource>
+    <!-- RELATEDPERSON -->
+    <resource>
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="MAY"/>
+      </extension>
+      <type value="Procedure"/>
```
Comment (dbojicic) [inline input/resources/au-erequesting-server.xml:341]: should be "RelatedPerson"

File: input/resources/au-erequesting-placer.xml [hunk 9]
```diff
@@ -83,5 +83,252 @@
...
+      </supportedProfile>
+      <documentation value="Placers **MAY** support the DocumentReference resource."/>
+      <!-- TBC 
+      <referencePolicy value="resolves">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHOULD"/>
+        </extension>
+      </referencePolicy> -->
+    </resource>
+    <!-- ENCOUNTER -->
```
Comment (dbojicic) [inline input/resources/au-erequesting-placer.xml:238]: Encounter included twice causes: Constraint failed: cpb-9: 'A given resource can only be described once per RESTful mode.
Remove this entry (MAY expectation).

File: input/resources/au-erequesting-server.xml [hunk 10]
```diff
@@ -105,5 +105,252 @@
...
+      </supportedProfile>
+      <documentation value="Servers **MAY** support the DocumentReference resource."/>
+      <!-- TBC 
+      <referencePolicy value="resolves">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHOULD"/>
+        </extension>
+      </referencePolicy> -->
+    </resource>
+    <!-- ENCOUNTER -->
```
Comment (dbojicic) [inline input/resources/au-erequesting-server.xml:260]: Encounter included twice causes: Constraint failed: cpb-9: 'A given resource can only be described once per RESTful mode.
Remove this entry (MAY expectation).

File: input/resources/au-erequesting-placer.xml [hunk 11]
```diff
@@ -83,5 +83,252 @@
         </extension>
       </referencePolicy> -->
     </resource>
+    <!-- COVERAGE -->
```
Comment (dbojicic) [inline input/resources/au-erequesting-placer.xml:86]: Not the line that needs to be fixed but commenting here to get an inline comment. 

In rest.description, please replace 'Server' with 'Placer".

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see inline comments

File: input/pagecontent/CapabilityStatement-au-erequesting-placer-intro.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/CapabilityStatement-au-erequesting-placer-intro.md]: fixed.

File: input/pagecontent/CapabilityStatement-au-erequesting-server-intro.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/CapabilityStatement-au-erequesting-server-intro.md]: fixed

File: input/pagecontent/CapabilityStatement-au-erequesting-server-intro.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/CapabilityStatement-au-erequesting-server-intro.md]: fixed.

File: input/pagecontent/CapabilityStatement-au-erequesting-placer-intro.md (hunk 4 above)
Comment (dt-r) [reply input/pagecontent/CapabilityStatement-au-erequesting-placer-intro.md]: fixed.

File: input/resources/au-erequesting-placer.xml (hunk 5 above)
Comment (dt-r) [reply input/resources/au-erequesting-placer.xml]: fixed.

File: input/resources/au-erequesting-server.xml (hunk 6 above)
Comment (dt-r) [reply input/resources/au-erequesting-server.xml]: fixed.

File: input/resources/au-erequesting-placer.xml (hunk 7 above)
Comment (dt-r) [reply input/resources/au-erequesting-placer.xml:319]: fixed.

File: input/resources/au-erequesting-server.xml (hunk 8 above)
Comment (dt-r) [reply input/resources/au-erequesting-server.xml]: fixed.

File: input/resources/au-erequesting-placer.xml (hunk 9 above)
Comment (dt-r) [reply input/resources/au-erequesting-placer.xml:238]: The erronous inclusion of Encounter as MAY is removed.

File: input/resources/au-erequesting-server.xml (hunk 10 above)
Comment (dt-r) [reply input/resources/au-erequesting-server.xml:260]: The erronous inclusion of Encounter as MAY is removed.

File: input/resources/au-erequesting-placer.xml (hunk 11 above)
Comment (dt-r) [reply input/resources/au-erequesting-placer.xml:86]: fixed.

**Review by dbojicic — APPROVED** [review]

**Pull request merged**